Sources/MapboxMaps/Annotations/AnnotationOrchestrator.swift

```diff
@@ -98,7 +117,7 @@ public class AnnotationOrchestrator {
             style: style,
             layerPosition: layerPosition,
             displayLinkCoordinator: displayLinkCoordinator,
-            clusterOptions: clusterOptions)
+            longPressGestureRecognizer: longPressGestureRecognizer)
```

Comment:
this should not be right, `clusterOptions` is added in https://github.com/mapbox/mapbox-maps-ios/pull/1651, could you cherry-pick changes from https://github.com/mapbox/mapbox-maps-ios-internal/pull/1294 to your code generator branch?